src/App.js

```diff
@@ -139,6 +143,27 @@ class App extends Component {
   };
 
   render() {
+
+    if ( 
+      this.props.availableOrganisations.length === 0 && 
+      this.props.isFetchingOrganisations === false &&
+      this.props.timesFetchedOrganisations > 0
+    ) {
+      const norolesMessage = this.props.intl.formatMessage({ id: "authorization.no_roles_message", defaultMessage: "Dear user, \nYou seem not to be in any organisations that can acces the management pages. \nTherefore you are redirected to the mainpage" });
+      alert(norolesMessage);
+      // should redirect to demo.lizard.net on prod
+      window.location = "/";
+    }
+
+    const currentHomeAppIcon = appIcons.find(icon => {
+      return window.location.href.includes(icon.linksTo.path)
+    });
+    if (currentHomeAppIcon && !doArraysHaveEqualElement(this.props.selectedOrganisation.roles, currentHomeAppIcon.requiredRoles)) {
+      const redirectMessage = this.props.intl.formatMessage({ id: "authorization.redirected_based_onrole", defaultMessage: "You do not have the rights to acces this data under the selected organisation. \nYou will be redirected" });
+      alert(redirectMessage);
```

Comment:
Yes good point. Problem is that if user hits cancel we would need put the selected organisation back the the previous one. That makes the change a bit bigger, but still doable. This would be a good next ticket, but it has probably lower priority.
https://nelen-schuurmans.atlassian.net/browse/FRNT-513